src/ProgressMeter/ProgressMeter.jl

```diff
@@ -39,16 +39,16 @@ function printprogress(
     barglyphs::BarGlyphs,
     tfirst::Float64,
     desc,
-    progress::Real,
+    progress::Union{Float64,Nothing},
```

Comment:
Let's reflect this to the docstring.

Reply:
updated. Thanks!

Reply:
THanks for the update. Actually, there is one more in

https://github.com/c42f/TerminalLoggers.jl/blob/9b26082d55dec100287862f1b2f4a9e89be8ec20/src/ProgressMeter/ProgressMeter.jl#L25

Reply:
Sorry, overlooked that...corrected now. Thanks!